### PR TITLE
removed "Action" section

### PR DESCRIPTION
**Pull request by MaratMussabekov**
as the only possible action is "quarantine" - see the note [here](https://docs.microsoft.com/powershell/module/exchange/antispam-antimalware/new-malwarefilterpolicy?view=exchange-ps)
#737

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/anti-malware-protection.md [hunk 1]
```diff
@@ -36,17 +36,7 @@ To submit malware to Microsoft, see [Submitting malware and non-malware to Micro
...
-- **Action**: Specifies what to do when a message is found to contain malware. The options are:
-
-  - Delete the message (this is the default value).
-
-  - Replace all attachments with a text file that contains this default text:
-
-    > Malware was detected in one or more attachments included with this email. All attachments have been deleted.
-
-  - Replace all attachments with a text file that contains the custom text you specify.
+When malware is detected, Exchange Online Protection allows the messages being delivered, but attachments that contain malware are quarantined. Anti-malware policies control notification options for malware detections. The important settings in anti-malware policies are:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/anti-malware-protection.md]:
```suggestion
When malware is detected, Exchange Online Protection allows the messages to be delivered, but attachments that contain malware are quarantined. Anti-malware policies control notification options for malware detection. The important settings in anti-malware policies are:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #737

Thank you.

cc: @kenwith

**Pull request merged**